src/main/git-elegant-completion

```diff
@@ -17,6 +17,11 @@ _git_elegant() {
         configure)
             COMPREPLY=($(compgen -W '--global --local' -- ${cur}) )
             return 0 ;;
+        check)
+            local data=("-a" "--all" "-u" "--unstaged" "-s" "--staged")
+            COMPREPLY=($(compgen -W "${data[*]}" -- ${cur}) )
```

Comment:
Is **`[*]`** mandatory after `data`?

Reply:
@extsoft Well, I tested and it doesn't work without `[*]`
In my opinion these two solutions both are ok:
```
check)
    COMPREPLY=($(compgen -W '"--all" "--unstaged" "--staged"' -- ${cur})
```
and
```
check)
    local data=("--all" "--unstaged" "--staged")
    COMPREPLY=($(compgen -W "${data[*]}" -- ${cur}) )
```
Which one is better in your opinion?

Reply:
@ivan-podorozhnyi The shortest.

Reply:
@extsoft already used it in my last commit :)